README.md

```diff
@@ -1 +1,177 @@
-# DeepStack-CameraUI
\ No newline at end of file
+# DeepStack-CameraUI
+DeepStack-CameraUI is a real-time computer vision web-application. Powered by Deepstack and Streamlit, this application performs realtime AI processing such as **object detection**, **face recognition**, **face detection** and **custom detection** on video frames from WebCam, Wired Camera or IP camera through an interactive web interface.
+
+## DeepStack-CameraUI Interface
```

Comment:
The 2  sections `DeepStack-CameraUI Interface` to `Deepstack-CameraUI Settings` should be after installation